### PR TITLE
Use SDL_KeyCode in LEGO1.dll

### PR DESCRIPTION
**Pull request by madebr**
I think this change deserves its own pr for testing purposes.

- Also use iniparser for getting mediaPath
- and replace a single isdigit with SDL_isdigit

### CODE REVIEW

File: ISLE/isleapp.cpp [hunk 1]
```diff
@@ -544,6 +550,10 @@ void IsleApp::LoadConfig()
 	strcpy(m_cdPath, cdPath);
 	MxOmni::SetCD(m_cdPath);
 
+	const char* mediaPath = iniparser_getstring(dict, "isle:mediapath", cdPath);
```
Comment (foxtacles) [inline ISLE/isleapp.cpp:553]: I think a better default would be `hdPath` instead of `cdPath`

**Comment by foxtacles** [comment]
In the registration book, we have an access violation when trying to type a name:

image

Probably just need to replace the `- 'A'` part

**Comment by foxtacles** [comment]
The debug triggers behind the "ogel" password don't work right now - I think you have to replace the `g_debugPassword` with an array (null terminated) of the corresponding SDL key codes.

**Comment by madebr** [comment]
> The debug triggers behind the "ogel" password don't work right now - I think you have to replace the `g_debugPassword` with an array (null terminated) of the corresponding SDL key codes.

Indeed. Long term, we should use a combination of SDL_Keycode and SDL_Scancode: SDL_Scancode for WASD and movement, and keycode for other cheats and/or text input.

**Comment by foxtacles** [comment]
Everything works now, thank you!

**Pull request merged**